src/porepy/models/fluid_property_library.py

```diff
@@ -0,0 +1,372 @@
+""""Module containing some constant heuristic fluid property implemenations.
+
+Most of the laws implemented here are meant for 1-phase, 1-component mixtures, using
+some fluid constants stored in the fluid's reference component.
+
+"""
+
+from __future__ import annotations
+
+from typing import Sequence, cast
+
+import porepy as pp
+
+from .protocol import PorePyModel
+
+__all__ = [
+    "FluidDensityFromPressure",
+    "FluidDensityFromTemperature",
+    "FluidDensityFromPressureAndTemperature",
+    "FluidMobility",
+    "ConstantViscosity",
+    "ConstantFluidThermalConductivity",
+    "FluidEnthalpyFromTemperature",
+]
+
+Scalar = pp.ad.Scalar
+ExtendedDomainFunctionType = pp.ExtendedDomainFunctionType
+
+
+class FluidDensityFromPressure(PorePyModel):
```

Comment:
This is the class that should be mixed in (before the FluidMixin) to get the standard mathematician's fluid? I suggest to expand the docstring with this information.

Reply:
Similarly, if I want to implement a new constitutive law (say, \rho=exp(cT*p)), I can use the current implementations as a template?

Reply:
Correct. I improved the module docstring and provided a simple example highlighting that.

And yes, the classes here are great examples for how to provide the callables for phase properties.

Reply:
Deleted comment on SurrogateFactory (which risked scaring off users), added a comment about how to implement other simple constitutive laws.